Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/FRFSTicketService.hs

```diff
@@ -241,7 +246,7 @@ getFrfsStations (_personId, mId) mbCity mbRouteCode mbStartStationCode vehicleTy
           possibleEndStops
     -- Return all the Stops
     _ -> do
-      stations <- B.runInReplica $ QStation.findByMerchantOperatingCityIdAndVehicleType merchantOpCity.id vehicleType_
+      stations <- B.runInReplica $ QStation.findByMerchantOperatingCityIdAndVehicleType _limit _offset merchantOpCity.id vehicleType_
```

Comment:
`_limit _offset` should Be passed as Nothing from UI Query.

Reply:
same reason, this query also expects limit and offset

Reply:
initially findAllByMerchantOperatingCityAndVehicleType was of type findAllWithKV, but since same was supposed to be used in Dashboard I changed it to findAllWithOptionsKV. Before implementing it, I confirmed from you also but I might have misinterpreted it.